src/h5web/dataset-visualizer/vis/matrix/MatrixVis.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { HDF5Value } from '../../../providers/models';
+import ValueCell from './ValueCell';
+import IndexedGrid from './IndexedGrid';
+import { AccessorFunc } from './utils';
+
+interface Props {
+  dims: number[];
+  data: HDF5Value;
+}
+
+function MatrixVis(props: Props): JSX.Element {
+  const { dims, data } = props;
+  const myGridSettings = {
+    cellSize: { width: 116, height: 32 },
+    rowCount: dims[0] + 1, // includes IndexRow
+    columnCount: (dims.length === 2 ? dims[1] : 1) + 1, // includes IndexColumn
+  };
```

Comment:
I reverted my use of `Context` for grid settings. I feel that passing it as props to the `Grid` that needs it and then, only the `cellSize` to the components needing it, is clearer.

Reply:
Put again in `Context` as it is not possible to pass props to `innerElementType`